SVS/model/network.py

```diff
@@ -470,6 +470,82 @@ def forward(self, encoder_out, align_phone, text_phone, pitch, beats):
 
         return out
 
+class Encoder_Postnet_combine(nn.Module):
+    """Encoder Postnet."""
+
+    def __init__(self, embed_size, singer_size):
+        """init."""
+        super(Encoder_Postnet_combine, self).__init__()
+
+        self.fc_pitch = nn.Linear(1, embed_size)
+        # Remember! embed_size must be even!!
+        self.fc_pos = nn.Linear(embed_size, embed_size)
+        # only 0 and 1 two possibilities
+        self.emb_beats = nn.Embedding(2, embed_size)
+        self.pos = module.PositionalEncoding(embed_size)
+
+        self.emb_singer = nn.Embedding(singer_size, embed_size)
+        self.fc_out = nn.Linear(2 * embed_size, embed_size)
+
+    def aligner(self, encoder_out, align_phone, text_phone):
+        """aligner."""
+        # align_phone = [batch_size, align_phone_length]
+        # text_phone = [batch_size, text_phone_length]
+        # align_phone_length( = frame_num) > text_phone_length
+        # batch
+        align_phone = align_phone.long()
```

Comment:
how about constructing a function here to uniform the alignment (I believe they are the same for all models)

Reply:
I will fix it.